src/main/resources/templates/db_scripts/schema.json

```diff
@@ -17,6 +17,11 @@
       "fromModuleVersion": "mod-invoice-storage-4.0.0",
       "withMetadata": true
     },
+    {
+      "tableName": "batch_voucher_export_configs",
+      "fromModuleVersion": "mod-invoice-storage-4.0.0",
+      "withMetadata": true
```

Comment:
Remember to add the foreign key to batch_voucher table if folio-org/mod-invoice-storage#58 is merged before that.

Reply:
Added a comment to https://github.com/folio-org/mod-invoice-storage/pull/58 to handle this.